File: src/main/java/com/starshootercity/abilities/incomplete/EnderParticles.java
package com.starshootercity.abilities.incomplete;

import com.starshootercity.abilities.Ability;
import net.kyori.adventure.key.Key;
import org.jetbrains.annotations.NotNull;

public class EnderParticles implements Ability {
    @Override
    public @NotNull Key getKey() {
        return Key.key("origins:ender_particles");
    }
}

File: core/src/main/kotlin/ru/turbovadim/abilities/EnderParticles.kt
package ru.turbovadim.abilities

import net.kyori.adventure.key.Key
import org.bukkit.Particle

class EnderParticles : ParticleAbility {

    override fun getKey(): Key {
        return Key.key("origins:ender_particles")
    }

    override val particle = Particle.PORTAL

    override val frequency: Int = 4
}
